corral/log/log.pony

```diff
@@ -1,29 +1,40 @@
-use "logger"
+"""
+# Logging in corral
 
-primitive CodeLogFormatter is LogFormatter
-  fun apply(msg: String, loc: SourceLoc): String =>
-    let file_name: String = loc.file()
-    let file_linenum: String  = loc.line().string()
-    let file_linepos: String  = loc.pos().string()
-    (recover String(file_name.size()
-      + file_linenum.size()
-      + file_linepos.size()
-      + msg.size()
-      + 4)
-    end)
-     .> append(file_name)
-     .> append(":")
-     .> append(file_linenum)
-     .> append(":")
-     .> append(file_linepos)
-     .> append(": ")
-     .> append(msg)
+Logging in corral uses the 'logger' package from the standard library, with
+custom implementations of the 'LogFormatter' interface.
+
+The API for 'logger' makes use of boolean short-circuiting in order to avoid the
+work of building strings that won't get logged. This can make log statements
+somewhat verbose:
+
+```
```

Comment:
can you pony codefence type?

Reply:
Done. It's `pony`, not `ponylang`, right?

Reply:
correct